src/neuroglancer/whats_new/whats_new.ts

```diff
@@ -24,15 +24,22 @@ const generateWhatsNew = (GHCommits: string[] = []) => {
 };
 
 export const findWhatsNew = async (viewer: Viewer) => {
-  let url =
-          `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`,
-      WNCommits = JSON.parse(localStorage.getItem('WNCommits') || '[]'), headers = {
-        'Content-Type': 'text/plain;charset=utf-8',
-      },
-      body = JSON.stringify({
-        path: updateFName
-        // since: (WNCommits.length) ? WNCommits[0].commit.author.date : void(0)
-      });
+  const url =
+      `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`;
```

Comment:
This URL requires a comment at least. It's not clear what is being delivered, who controls it, and why it must be a third party URL.

Reply:
It kind of looks like a script from the `exec` part.